rfd/0086/multiprocess.md

```diff
@@ -117,54 +131,101 @@ ContainerPilot hasn't eliminated the complexity of dependency management -- that
 
 That being said, a more expressive configuration of event handlers may more gracefully handle all the above situations and reduce the end-user confusion. Rather than surfacing just changes to dependency membership lists, we'll expose changes to the overall state as ContainerPilot sees it.
 
-ContainerPilot will provide the following events:
+ContainerPilot will provide events and each service can opt-in to having a `start` condition on one of these events. Because the life-cycle of each service triggers new events, the user can create a dependency chain among all the services in a container (and their external dependencies). This effectively replaces the `preStart`, `preStop`, and `postStop` behaviors.
```

Comment:
In a sidebar discussion we've discussed that `when` is a more ergonomic than `start` for jobs that run more than once on on multiple triggered events. This gives us syntax like:

- `when: "startup"`
- `when: "myPreStart exitSuccess timeout 60s"`
- `when: "myDb healthy"`
- `when: "myDb stopped"`